cli/config_file.go

```diff
@@ -155,6 +162,93 @@ func SettingsFileReadWrite(state *Settings) *Settings {
 	return settings
 }
 
+// Check prerequisites
+func CheckRequirements() {
+	// getenv
+	envVarValue := os.Getenv("DOCKER_HOST")
+
+	if envVarValue != "" {
+		socketurl, err := url.ParseRequestURI(envVarValue)
+		if err != nil {
+			log.Fatalf("error: %v", err)
+		}
+
+		if socketurl.Host != "" {
+			log.Fatalf("error: DOCKER_HOST variable shouldn't have host")
+		}
+
+		if socketurl.Scheme != "unix" {
+			log.Fatalf("error: DOCKER_HOST variable should contain a valid unix socket")
+		}
+	} else {
+		// We cannot assume unix:///var/run/docker.sock on Mac/Win platforms, we let Docker SDK does its magic :)
+		log.Println("\033[33mwarning: DOCKER_HOST environmental variable is empty or not set.")
```

Comment:
![image](https://user-images.githubusercontent.com/9247788/191971294-588d98e6-e26b-4cf9-bc2d-9031bb5133e4.png)

Stop coloring every line ending or just do it in one Println.

Reply:
now both lines have start/stop characters in them regarding to coloring